toolkit/job-analysis/agenda-for-job-analysis-workshop-day-1.md

```diff
@@ -5,89 +5,172 @@ section: hiring-phases
+        Explain Proficiency Levels
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">1 hour 15 min</th>
+      <td>
+        Create proficiency levels for each competency
         <ul>
-          <li>Use examples if needed from DOI and USDS</li>
-          <li>Definition should be 1-3 sentences, not a list of job tasks</li>
+          <li>Break into groups as before and divide competencies between groups</li>
+          <li>Reference examples and have proficiencies build on each other</li>
         </ul>
       </td>
     </tr>
     <tr>
-      <th scope="row">2:00 pm</th>
+      <th scope="row">45 min</th>
       <td>
-        Understanding Proficiency Levels (5 min)
+        Discuss and finalize proficiency levels with group
+        <ul>
+          <li>Time required will depend on SME alignment</li>
+        </ul>
       </td>
     </tr>
     <tr>
-      <th scope="row">2:05 pm</th>
+      <th scope="row">15 minutes</th>
       <td>
-        Create proficiency levels for each competency (1 hour 15 min)
+        Confidentiality agreements
         <ul>
-          <li>Break into 2 groups and divide competencies between groups</li>
+          <li>SMEs sign privacy agreement on testing materialsw</li>
+            <ul>
+              <li>HR should talk about importance of confidentiality test materials</li>
+              <li>Use this time to ensure everyone knows who is saving/storing the test materials</li>
+            </ul>
         </ul>
       </td>
     </tr>
     <tr>
-      <th scope="row">3:20 pm</th>
+      <th scope="row">30 min</th>
+      <td>
+        Review job tasks against competencies
+          <ul>
+            <li>Only needed if you are on day 2 of the workshop at this point and need to refresh SMEs</li>
+          </ul>
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">1 hour 30 minutes</th>
+      <td>
+        Resume review exercise and competency/proficiency refinement
+        <ul>
+          <li>This is the most important exercise so far - the group will check its work to date</li>
+          <li>Hand SMEs sample resumes (ideally, difficult resumes) and have them review against the current drafts</li>
+          <li>Privately record SME responses, then show results</li>
+          <li>Group should go over results and iterate on materials to achieve:</li>
+          <ul>
+            <li>Clarity on if any competancies have more weight than others</li>
+            <li>Clarify on recency relevance for any competencies</li>
+            <li>Clarity on number of resumes pages to review (2-5)</li>
+          </ul>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">15 minutes</th>
+      <td>
+        Overview of assessments and tentative selections of assessment types
+        <ul>
+          <li>USDS suggests a first-round written assessment and second round oral interview</li>
+          <li>You can also use two interviews: one on breadth, one on depth</li>
```

Comment:
explain further- the two interviews need to measure different things since we're eliminating applicants between the two rounds. for example breadth vs depth of knowledge or core competencies then parenthetical competencies.

Reply:
updated